flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -95,6 +95,9 @@ def _get_package_source(self, default_version=None):
         unsupported={backends.LOOPBACK},
         reason="Does not maintain compute_instance_id across restarting "
                "flocker (and didn't as of most recent release).")
+    @skip_backend(
+        unsupported={backends.GCE},
+        reason="GCE was not available during the most recent release.")
```

Comment:
Done.

I suppose I also could have done some fancy skip if version number is less than blah and backend is GCE... whatever, future Marcus won't mind having to do this later.